src/index.js

```diff
@@ -207,10 +208,26 @@ function sanitizeHtml(html, options, _recursing) {
               }
             }
             if (name === 'iframe' && a === 'src') {
-              var whitelistedUrls = options.allowedIframeDomains.filter(function(url) {
-                return value.includes(url);
-              })
-              if (!whitelistedUrls.length) {
+              try {
+                var parsed = url.parse(value)
+                var whitelistedDomains = []
+                each(parsed, function(val, prop) {
```

Comment:
This each loop is redundant now, you are only interested in: `parsed.hostname`, so just check that property.

Reply:
Removed.